apps/meteor/app/utils/lib/slashCommand.ts

```diff
@@ -30,14 +31,16 @@ export const slashCommands = {
 		previewer,
 		previewCallback,
 		appId,
+		description = '',
 	}: ISlashCommandAddParams<T>): void {
 		this.commands[command] = {
 			command,
 			callback,
 			params: options.params,
-			description: options.description,
+			description: options.description || description,
```

Comment:
instead of creating a new `description` field that seems like isn't being used anywhere, could you update it to be `options.description || ''` or define it as default on the initialization of the `options` field?

Reply:
I believe that this will not be possible because, depending of the context, description comes from options or not